lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -11,6 +11,14 @@ create:
 destroy:
   config_set: "no vlan %s"
 
+mapped_vni:
+  /N(3|9)/:
+    kind: int
+    config_get: 'show running vlan'
+    config_get_token: ['/^vlan <vlan>$/', '/^vn-segment (\d+)$/']
+    config_set: ['vlan <vlan>', '<state> vn-segment <vni> ; end']
```

Comment:
Ok.. please add a comment then to indicate that the end is needed for this specific property and why.